tests/test_algorithm.py

```diff
@@ -387,6 +444,22 @@ def test_order_methods(self):
             )
             algo.run(self.df)
 
+    def test_order_methods_for_future(self):
```

Comment:
Can we do more than just assert that this doesn't crash?  What do we actually expect to happen here?